addons/account/wizard/setup_wizards.py

```diff
@@ -116,7 +116,7 @@ def _compute_linked_journal_id(self):
 
     def default_linked_journal_id(self):
         default = self.env['account.journal'].search([('type', '=', 'bank'), ('bank_account_id', '=', False)], limit=1)
-        return default and default[0].id
+        return default[0].id if default else False
```

Comment:
This is equivalent and slightly simpler:

```suggestion
        return default[:1].id
```